usr/local/www/diag_ipsec_leases.php

```diff
@@ -47,85 +49,103 @@
 $shortcut_section = "ipsec";
 include("head.inc");
 
-$mobile = ipsec_dump_mobile();
+if(DEBUG)  { // Dummy data for layout testing. REMOVE before production.
+	$mobile = array('pool' => array( 0 => array( 'name' => '172.27.113.0/24',  'size' => '254',	 'usage' => '1', 'online' => '1',
+												 'lease' => array(0 => array('id' => 'Firstly', 'status' => 'Ready', 'host' => '192.168.1.43'),
+																 1	=> array('id' => 'Who', 'status' => 'Ready', 'host' => '10.255.255.7'))),
+									 1 => array( 'name' => '192.48.111.0/24',  'size' => '254',	 'usage' => '1', 'online' => '1',
+												 'lease' => array(0 => array('id' => 'Second', 'status' => 'Ready', 'host' => '192.168.8.20'))),
+									 2 => array( 'name' => '172.48.222.0/24',  'size' => '254',	 'usage' => '1', 'online' => '1'),
+									 3 => array( 'name' => '192.48.111.0/24',  'size' => '254',	 'usage' => '1', 'online' => '1',
+												 'lease' => array(0 => array('id' => 'Second', 'status' => 'Ready', 'host' => '192.168.8.20')))
+									 ));
+}
+else
+	$mobile = ipsec_dump_mobile();
+
+$tab_array = array();
+$tab_array0 = array(gettext("Overview"), false, "diag_ipsec.php");
+$tab_array1 = array(gettext("Leases"), true, "diag_ipsec_leases.php");
+$tab_array2 = array(gettext("SAD"), false, "diag_ipsec_sad.php");
+$tab_array3 = array(gettext("SPD"), false, "diag_ipsec_spd.php");
+$tab_array4 = array(gettext("Logs"), false, "diag_logs.php?logfile=ipsec");
+display_top_tabs($tab_array);
+
+if (isset($mobile['pool']) && is_array($mobile['pool'])) {
+?>
+	<div class="table-responsive">
+		<table class="table table-hover table-condensed">
+			<thead>
+				<tr>
+					<th><?=gettext("Pool")?></th>
+					<th><?=gettext("Usage")?></th>
+					<th><?=gettext("Online")?></th>
+					<th><?=gettext("ID")?></th>
+					<th><?=gettext("Host")?></th>
+					<th><?=gettext("Status")?></th>
+				</tr>
+			</thead>
+			<tbody>
+<?php
+			foreach($mobile['pool'] as $pool) {
```

Comment:
I'd favor foreach:/endforeach and if:/endif in this file. If you put those on one line it'd better readable

```php
<?php foreach($mobile['pool'] as $pool): ?>
```